packages/storefront/src/views/Page.vue

```diff
@@ -56,6 +73,16 @@
 
     private async mounted(): Promise<void> {
       await this.loadComponents();
+
+      onPrefersColorSchemeChange(this, () => {
+        if (this.storefrontTheme === 'auto') {
+          this.$forceUpdate();
```

Comment:
Why is forceUpdate necessary here, isn't it already handled within CycleStorefrontTheme component?

Reply:
Otherwise wrong class might be applied since `getSkeletonTheme()` isn't called again.